Packs/CortexAttackSurfaceManagement/Scripts/RankServiceOwners/RankServiceOwners_test.py

```diff
@@ -1,87 +1,64 @@
 import demistomock as demisto  # noqa: F401
```

Comment:
There's a docstring format for tests that I've had to use in this codebase before -- something about When/Given/Then or the like. you'll probably be asked to refactor to this format (you can find the format in a parallel test file) -- it might be worth confirming this and then getting ahead of it.

Reply:
+1 Please you docstrings for all functions.